sdks/python/apache_beam/io/gcp/datastore/v1/helper_test.py

```diff
@@ -28,9 +28,12 @@
 from mock import MagicMock
 
 # pylint: disable=ungrouped-imports
-from apache_beam.io.gcp.datastore.v1 import fake_datastore
-from apache_beam.io.gcp.datastore.v1 import helper
-from apache_beam.testing.test_utils import patch_retry
+try:
```

Comment:
In places where we skip Datastore imports, let's add a comment, something like:
`# TODO(BEAM-4543): googledatastore dependency does not work on Python 3.`